NitroxServer-Subnautica/GameLogic/Entities/Spawning/SubnauticaEntitySpawnPointFactory.cs

```diff
@@ -20,21 +20,23 @@ public override List<EntitySpawnPoint> From(AbsoluteEntityCell absoluteEntityCel
                     List<EntitySlot.Type> slotTypes = SlotsHelper.GetEntitySlotTypes(entitySlotData);
                     List<string> stringSlotTypes = slotTypes.Select(s => s.ToString()).ToList();
 
-                    spawnPoints.Add(
-                        new EntitySpawnPoint(absoluteEntityCell,
-                                             entitySlotData.localPosition,
-                                             entitySlotData.localRotation,
-                                             stringSlotTypes,
-                                             entitySlotData.density,
-                                             entitySlotData.biomeType.ToString())
-                    );
+                    new EntitySpawnPoint(absoluteEntityCell,
```

Comment:
It is very ugly to hide that the constructor adds the item to an internal static dictionary on `gameObject.Parent` here. Any ideas for a slight refactor?